k8s-openapi-codegen-common/src/swagger20/mod.rs

```diff
@@ -71,6 +71,7 @@ impl<'de> serde::Deserialize<'de> for Spec {
 			#[serde(default)]
 			parameters: Vec<std::sync::Arc<Parameter>>,
 			responses: std::collections::BTreeMap<String, InnerResponse>,
+			#[serde(default)]
```

Comment:
This will create `("",)` when the tag isn't there. It'd be better to make this `tags: Option<(String,)>,` so that it's handled correctly.

Reply:
Done.